CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [1.0.8] - 2024-03-21
+### Changed
+- Dashboard now logs when a call has started to prevent duplicate calls
+
```

Comment:
plus bugfix when going to different day right?

Reply:
added it